recipes/qsimcirq/meta.yaml

```diff
@@ -0,0 +1,64 @@
+{% set name = "qsimcirq" %}
+{% set version = "0.19.0" %}
+
+package:
+  name: {{ name|lower }}
+  version: {{ version }}
+
+source:
+  url: https://github.com/quantumlib/qsim/archive/refs/tags/v0.19.0.tar.gz
+  sha256: 81101fe9e6356e54125f703956e787aa366717c1e6e77aea794879b53c84ff60
+
+build:
+  skip: true  # [py2k]
+  skip: true  # [win]
+  number: 0
+  script: {{ PYTHON }} -m pip install . -vvv --no-deps --no-build-isolation
+
+requirements:
+  build:
+    - {{ compiler('c') }}
+    - {{ compiler('cxx') }}
+    - pybind11-abi
+    - llvm-openmp  # [osx]
+    - libgomp      # [linux]
+    - openmpi
+    - cmake >=3.27
+  host:
+    - python
+    - pybind11
+    - make
+    - setuptools >=45
+    - wheel
+    - pip
+  run:
+    - python
+    - pybind11
+    - absl-py
+    - cirq-core
+    - typing_extensions
+    - {{ pin_compatible('numpy') }}
```

Comment:
I don't see a numpy in host or build. This needs to be in both.

Reply:
Thanks! Added in https://github.com/conda-forge/staged-recipes/pull/24504/commits/10dedc4a4c03c4c99cba8adbf0a1e0662d37bd33.